src/domain/app/i18n/en.json

```diff
@@ -464,7 +464,15 @@
         "tomorrow": "Tomorrow at {{time}}"
       }
     },
-    "placeholderSearch": "Search",
+    "search": {
+      "placeholderSearch": "Search",
+      "labelSearch": "Search",
+      "labelPlaces": "Places",
+      "placeholderPlaces": "Search places...",
+      "placesToggleButtonAriaLabel": "Open dropdown",
+      "placesClearButtonAriaLabel": "Delete all places",
```

Comment:
Something like "Clear place selections" would be better. Delete is like really getting rid of them :)

Reply:
True :D